tmt/steps/report/reportportal.py

```diff
@@ -0,0 +1,62 @@
+import io
+import os
+import zipfile
+
+import junit_xml
+import requests
+
+import tmt.steps.report
+import tmt.utils
+
+from . import junit
+
+
+class ReportPortal(tmt.steps.report.ReportPlugin):
+    """
+    Send results in JUnit format to ReportPortal instance.
+
+    Requires environment variables TMT_REPORT_PORTAL_URL and TMT_REPORT_PORTAL_TOKEN.
+    """
+
+    _methods = [tmt.steps.Method(name="reportportal", doc=__doc__, order=60)]
+
+    def go(self) -> None:
+        """
+        Read executed tests, prepare junit, compress it to a zip zile and
+        send it to ReportPortal instance.
+        """
+        for env_var in ("TMT_REPORT_PORTAL_URL", "TMT_REPORT_PORTAL_TOKEN"):
```

Comment:
I for one would expect at least a basic support for these "options" - I suppose envvars are the preferred way under TF rule, and in the case of token it makes sense to prefer envvar instead of the CLI option, but is there a reason to *not* have `--report-portal-url=...` or even `report-portal-url: ...` key in tmt specification? Not having options or keys would make this plugin very special among its peers.